docs/installation.md

```diff
@@ -55,15 +58,17 @@ First we'll need to clone the operator:
 ```
 
 We try to maintain all resources created/required by ripsaw in the namespace
```

Comment:
Can you add that we did this to support introduction into OperatorHub so people don't think we randomly picked it.